lib/jazzy/jazzy_markdown.rb

```diff
@@ -13,10 +13,9 @@ def header(text, header_level)
                       .sub(/^-/, '')
                       .sub(/-$/, '')
 
-      "<a href='##{text_slug}' class='anchor' aria-hidden=true>" \
-        '<span class="header-anchor"></span>' \
```

Comment:
Don’t mind me – just pointing out something that came to mind. 😅 I don’t know if any themes actually do this. It would be feasible for a third-party theme to reimplement this functionality in JavaScript anyhow.